depends/packages/libevent.mk

```diff
@@ -14,6 +16,7 @@ define $(package)_set_vars
   $(package)_config_opts_release=--disable-debug-mode
   $(package)_config_opts_linux=--with-pic
   $(package)_config_opts_android=--with-pic
+  $(package)_cppflags_mingw32=-D_WIN32_WINNT=0x0601
```

Comment:
I don't see how this can work... evutil.c `#undef _WIN32_WINNT`

Reply:
It's only [undef'd](https://github.com/libevent/libevent/blob/45c3fc29fb2240dbaa066c971f3fd59861293305/evutil.c#L43) after the inclusion of [`ws2tcpip.h`](https://github.com/libevent/libevent/blob/45c3fc29fb2240dbaa066c971f3fd59861293305/evutil.c#L33) which is where the `AI_*` definitions are.

In the `ws2tcpip.h` header on my machine I've got:
```c
#define AI_PASSIVE                  0x00000001
#define AI_CANONNAME                0x00000002
#define AI_NUMERICHOST              0x00000004
#if (_WIN32_WINNT >= 0x0600)
#define AI_NUMERICSERV              0x00000008
#define AI_ALL                      0x00000100
#define AI_ADDRCONFIG               0x00000400
#define AI_V4MAPPED                 0x00000800
#define AI_NON_AUTHORITATIVE        0x00004000
#define AI_SECURE                   0x00008000
#define AI_RETURN_PREFERRED_NAMES   0x00010000
#endif
```

Reply:
Oh, weird. That seems like it's probably a bug, but not our problem.

Reply:
You might want to add a comment here in the makefile about this is defined, and why it works.

Reply:
@laanwj I've added a comment here. These kinds of details probably deserve a separate document in depends (we could also add some of the macOS stuff there as well). I can follow up by writing up some more extensive docs.